xbmc/addons/AddonManager.cpp

```diff
@@ -218,57 +217,14 @@ VECADDONS CAddonMgr::GetAvailableUpdates() const
   VECADDONS updates;
   VECADDONS installed;
   VECADDONS all_addons;
-  m_database.GetRepositoryContent(all_addons);
-  std::map<std::string, AddonPtr> latestPrivateVersions;
-  std::map<std::string, AddonPtr> latestOfficialVersions;
+  CAddonRepos RepoInfos(*this);
```

Comment:
Variable should be call `addonRepos`. Bar names should never start with a capital latter.

Reply:
done